src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/Models/ListViewViewModel.cs

```diff
@@ -22,7 +22,7 @@ public class ListViewViewModel : ViewModelBase
 		private object[] _randomItems = new object[0];
 		private string _newInput = "1,1,2,2,3,3";
 		private List<string> _sampleItemsGenerated = new List<string> { "1", "1", "2", "2", "3", "3" };
-		private List<string> _singleItemList = new List<string> { "1" };
+		private readonly List<string> _singleItemList = new List<string> { "1" };
```

Comment:
it seems odd to have a readonly in the middle
might as well convert this into a get-only property, since we can now: `SingleItemList { get; } = ...`